config.json.sample

```diff
@@ -1,6 +1,7 @@
 {
   "zk": "zk://127.0.0.1:2181/mesos",
   "masters": ["127.0.0.1:5050"],
+  "stateTimeoutSeconds": 300,
```

Comment:
I can't think of a good reason, probably because there isn't one.

On Thu, Nov 12, 2015 at 12:06 PM, Sargun Dhillon <notifications@github.com>
wrote:

> In config.json.sample
> <https://github.com/mesosphere/mesos-dns/pull/341#discussion_r44684054>:
>
> > @@ -1,6 +1,7 @@
> >  {
> >    "zk": "zk://127.0.0.1:2181/mesos",
> >    "masters": ["127.0.0.1:5050"],
> > +  "stateTimeoutSeconds": 300,
>
> One of the questions I have is why we didn't use strings, and
> time.Duration / time.ParseDuration? @jdef <https://github.com/jdef> do
> you know?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/mesos-dns/pull/341/files#r44684054>.
>